examples/account/account_id_populate_from_mirror.py

```
@@ -0,0 +1,178 @@
+            .add_hbar_transfer(evm_account_id, Hbar(1).to_tinybars())
+            .add_hbar_transfer(client.operator_account_id, Hbar(-1).to_tinybars())
+            .execute(client)
+        )
+
+        receipt = (
+            TransactionGetReceiptQuery()
+            .set_transaction_id(transfer_tx.transaction_id)
+            .set_include_children(True)
+            .execute(client)
+        )
+    except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)